src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -451,12 +451,347 @@ The following changes must be performed:
+
+When this generic information is configured for a _Task_, a ProActive Node will be dynamically deployed on the cluster to execute the Task. This ProActive Node will be associated with the Task and will only accept this Task execution. When the Task terminates, the ProActive Node will also be terminated.
+
+When this generic information is configured for a _Job_, all Tasks contained in the Job will execute on the cluster. Nodes will be created dynamically to execute Tasks of this Job, with a similar Task/Node association.
+
+The behavior of the Meta-Scheduling feature is summarized on the following diagram (example for PBS integration):
+
+image::NativeScheduler.png[PBS Integration, , title="PBS Integration"]
+
+The following paragraphs explain how to configure the Native Scheduler Infrastructure, Native Scheduler Policy, Native Scheduler Scheduling Policy and execute ProActive tasks using a Native Scheduler Node Source.
+
+==== Native Scheduler Node Source Configuration
+
+Using the <<_glossary_rm_web_interface,*Resource Manager Web Interface*>>, you can create a <<_glossary_node_source,*Node Source*>> used to aquire <<_glossary_proactive_node,*ProActive Nodes*>> from a Native Scheduler.
+
+From The drop down menu, the *NativeSchedulerInfrastructure* and *NativeSchedulerPolicy* must be selected.
+
+Here is an explanation of the node source parameters:
+
+ * `Name` : you should name the node source accordingly to your infrastructure, it can be the cluster name, head node name, or native scheduler name (PBS, SLURM), etc.
+
+===== NativeSchedulerInfrastructure
+
+ * `RMCredentialsPath` : point to a file which contains the credentials of an administrator user which will own the node source. The ProActive Scheduler Server release contains two admin users credentials files : `config/authentication/rm.cred` and `config/authentication/admin_user.cred`
+ * `NSFrontalHostAddress` : the host name or IP address of the cluster head node.
+ * `NSSchedulerHome` : the location of the shared ProActive installation on cluster nodes (cluster nodes must be able to access ProActive libraries in order to start ProActive Node). Example `/opt/proactive/activeeon_enterprise-node-linux-x64-8.1.0`.
+ * `javaHome` : similarly, cluster nodes must be able to access the java command in order to start ProActive Nodes. ProActive installation includes a Java Runtime Environment under the `jre` subfolder. Example: `/opt/proactive/activeeon_enterprise-node-linux-x64-8.1.0/jre`.
+ * `jvmParameters` : additional options which can be passed to the java command.
+ * `sshOptions` : additional options which can be passed to the ssh command used to connect to NSFrontalHostAddress.
+ * `NSNodeTimeoutInSeconds` : timeout to wait for the deployment of ProActive Nodes on the cluster. As it the time needed to deploy ProActive Nodes depends on the cluster load, this timeout should be a large value.
```

Comment:
"`As it the time`" --> typo: `As the time`
Also, we can say right now what happens if the timeout is reached.

Reply:
ok